back-end/api/controllers/analyseProject.controller.js

```diff
@@ -0,0 +1,28 @@
+import AnalyseProjectService from '../../services/analyseproject.service.js'
```

Comment:
Great idea on making a separate controller and route for analyzing projects instead of just adding it to projects.controller/routes!